index.js

```diff
@@ -29,10 +29,15 @@ app.use('/prompt-sheet-for-working-offline', (req, res) => {
 });
 
 app.use((req, res, next) => {
-  // Overrides the Gov UK template to add alt tags to images
-  res.locals.partials['govuk-template'] = path.resolve(__dirname, './govuk_template');
   // Set HTML Language
   res.locals.htmlLang = 'en';
+  // Set feedback and footer links
+  res.locals.feedbackUrl = '/feedback';
+  res.locals.footerSupportLinks = [
+    { path: '/cookies', property: 'base.cookies' },
+    { path: '/terms-and-conditions', property: 'base.terms' },
+    { path: '/accessibility', property: 'base.accessibility' },
+  ];
```

Comment:
can you explain this a bit more?

Reply:
Within the custom layout that I removed the footer support links were hard coded in. This code provides footer support links to the locals which sets the links in the footer using the below template block in the default layout file.

<img width="538" alt="Screenshot 2021-02-26 at 11 22 47" src="https://user-images.githubusercontent.com/61828376/109294341-f5a02100-7824-11eb-8e58-5ff344fcf4c8.png">